components/engine_panic/src/misc.rs

```diff
@@ -17,4 +17,8 @@ impl MiscExt for PanicEngine {
     ) -> Result<()> {
         panic!()
     }
+
+    fn get_approximate_memtable_stats_cf(&self, cf: &str, range: &Range) -> Result<(u64, u64)> {
```

Comment:
How about document the function?

Reply:
I'll go ahead and add some docs, though note that this is a function I've just ported from elsewhere.

At some later phase I intend to go over the entire engine_traits crate and document everything.

Reply:
Oh, this function already had some docs that I didn't bring over, so good thing you did point it out.